### ISSUE TITLE
HGETALL and HINCRBY functions not declared in typescript types

### ISSUE THREAD

**Issue by AmitSrivastava89** [luin/ioredis#1096]
Though HGETALL and HINCRBY are implemented in Cluster class but in the types node module package its not defined. I had to do a hack to fix it by extending and just declaring the methods. Please fix it. The hack I did was:

```
interface MyCluster extends Cluster {
    hgetall(key: IORedis.KeyType): Promise<Record<string, string>>;
    hincrby(key: IORedis.KeyType, field: string, increment: number): Promise<number>;
}
```

**Comment by meetr73** [comment]
Did you add your custom interface in custom types?

**Comment by alavers** [comment]
I've opened a [PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43906) to add these and many other missing command definitions. The PR changes `Cluster` such that it inherits commands from a common `Commands` interface, which should hopefully make these definition gaps a thing of the past.

**Comment by alavers** [comment]
Should be fixed now with @types/ioredis 4.16.0

**Comment by luin** [comment]
ioredis v5 provides official TypeScript declarations so closing this. Sorry for the late response!
